tests/Type/UnitPriceTest.php

```diff
@@ -1,24 +1,31 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Type;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Type;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).